README.agda

```diff
@@ -43,40 +43,54 @@ module README where
 --     properties needed to specify these structures (associativity,
 --     commutativity, etc.), and operations on and proofs about the
 --     structures.
+
 -- • Axiom
---     The consequences of assuming various additional axioms
---     e.g. uniqueness of identity of proofs, function extensionality,
---     excluded middle.
+--     Types and consequences of various additional axioms not
+--     provable in Agda, e.g. uniqueness of identity of proofs,
```

Comment:
Nitpicking, but UIP is of course provable in Agda, and function extensionality is provable using `--cubical`...
How about simply

Types and consequences of various additional axioms, e.g. uniqueness of identity of proofs,  function extensionality and excluded middle.